src/metabase/server/handler.clj

```diff
@@ -42,11 +43,21 @@
        (name kkey))
      response output-stream)))
 
+
+(defn wrap-locale-binding [handler]
```

Comment:
Note for people who will see this:
I just hacked this together as POC, I'm not a clojure developer. I'm now realizing the `let` does absolutely nothing 🤷